src/sentry/api/event_search.py

```diff
@@ -67,7 +111,17 @@ def snuba_name(self):
 
 
 class SearchValue(namedtuple('SearchValue', 'raw_value')):
-    pass
+
+    @property
+    def value(self):
+        if self.is_wildcard():
+            return translate(self.raw_value)
+        return self.raw_value
+
+    def is_wildcard(self):
+        if not isinstance(self.raw_value, six.string_types):
+            return False
+        return bool(WILDCARD_CHARS.search(self.raw_value))
```

Comment:
@mattrobenolt i decided to do this here instead of in the grammar because i felt like having to have: wildcard quoted pattern, wildcard unquoted pattern, non-wildcard quoted pattern, non-wildcard unquoted pattern would be kind of a lot of repetition. let me know if you think that's a bad idea though.

Reply:
I think this is fine.